components/FeedbackModal.js

```diff
@@ -121,27 +121,35 @@ export default function FeedbackModal() {
       }
       const feedbackData = {
         from: formData.from.trim(),
-        subject: formData.subject.trim(),
         token: turnstileToken,
       }
 
+      const cleanSubject = formData.subject.trim()
+
       switch (formData.subject) {
         case profileSubject:
           feedbackData.message = `Profile ID: ${formData.profileId}\n\n${formData.message}`
-          feedbackData.subject = `${formData.subject} - ${formData.profileId}`
+          feedbackData.subject = `${cleanSubject} - ${formData.profileId}`
           break
         case submissionSubject:
           feedbackData.message = `Venue ID: ${formData.venueId}\nSubmission ID: ${formData.submissionId}\n\n${formData.message}`
+          feedbackData.subject = `${cleanSubject}${
```

Comment:
It doesn't seem like you addressed the issue I had with this change I think it would be better if it was

```suggestion
          feedbackData.subject = `Submission ${
```